<?xml version="1.0"?>

<launch>
  <arg name="uri1" default = "radio://0/120/2M/E7E7E7E704" />

<include file="$(find crazyflie_driver)/launch/crazyflie_server.launch">
  </include>

  <group ns="cf1">
    <!--comment-->
    <node pkg="crazyflie_driver" type="crazyflie_add" name="crazyflie_add" output="screen">
      <param name="uri" value="$(arg uri1)" />
      <param name="tf_prefix" value="cf1" />
      <param name="enable_logging" value="True" />
      <param name="enable_logging_temperature" value="False" />
      <param name="enable_logging_magnetic_field" value="False" />
      <param name="enable_logging_pressure" value="False" />
      <param name="enable_logging_imu" value="False" />
      <param name="enable_logging_battery" value="False" />
      <!-- <param name="crazyflie/ctrlMel/kp_z" value="0.8"/>  -->
       <rosparam>
        genericLogTopics: ["log_ranges" ]
        genericLogTopicFrequencies: [20]
        genericLogTopic_log_ranges_Variables: ["range.front", "range.back", "range.up", "range.left", "range.right" , "range.zrange" ]
      </rosparam> 
    </node>



  </group>



    <node name="crazyflie_demo_hover" pkg="crazyflie_demo" type="test_high_level.py" output="screen">
    </node>

</launch>
